cdap-ui/server/express.js

```diff
@@ -239,7 +239,7 @@ function makeApp (authAddress, cdapConfig) {
       function (req, res) {
         var apptype = req.params.apptype;
         var config = {};
-        var fileConfig = {}
```

Comment:
I guess this formatting is already done in previous PRs
https://github.com/caskdata/cdap/commit/6286113c14a834b21ed82c0f86d0d98d2ccdc121